CMakeLists.txt

```diff
@@ -65,6 +65,8 @@ if(ENABLE_COVERAGE)
     include(cmake/coverage.cmake)
 endif()
 
+set(CMAKE_INTERPROCEDURAL_OPTIMIZATION_RELEASE ${ENABLE_LTO})
```

Comment:
it ensures that all binaries (even tests) are build with LTO support.
Probably, we don't need it as it adds extra time to link stage.

I suppose we need to build with LTO only targets which later become OpenVINO release archive, while tests and other tools can be build w/o LTO.

Reply:
From what I saw, some test directories have CMakeList files that specifically turn LTO off.

Reply:
I suppose those are unit tests?
They fix LTO and non-LTO code and that us why manual setting are required.